src/components/shared/Entity/LogDialog.tsx

```diff
@@ -29,6 +52,7 @@ function LogDialog({ open, token, actionComponent, title }: Props) {
             >
                 <ErrorLogs logToken={token} defaultOpen />
```

Comment:
`ErrorLogs` should likely be renamed since it is being used to display standard logs and error logs.